### DISCUSSION TITLE
Failed to import

### DISCUSSION THREAD

**Discussion by hubble63** [Homebrew/discussions#3162] [Getting started]
Hello @ all,

I use MacOS 10.13.6 and I would like to start my own private guthub.
I´m an absolute beginner and have no one Idea how to solve this problem.

So I tried to install Homebrew by https://brew.sh/

After download all the packages I go 582 errors started with this:

//---
Error: Failed to import: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/cask.rb
cask: Calling bottle :unneeded is disabled! There is no replacement.
Please report this issue to the homebrew/core tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/cask.rb:9
---//

and ended with

//---
> Error: Failed to import: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/gocryptfs.rb
> gocryptfs: Unsupported special dependency :osxfuse
> Error: Failed to import: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/tomcat.rb
> tomcat: Calling bottle :unneeded is disabled! There is no replacement.
> Please report this issue to the homebrew/core tap (not Homebrew/brew or Homebrew/core):
>   /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/tomcat.rb:9
--//

I´m afraid,  I´m in the wrong discussion:
"Please report this issue to the homebrew/core tap"
Where I can find this tap?

Please let me understand where is the problem and what I have to do to solve it.

**Comment by SMillerDev** [comment]
Your homebrew is very outdated. Try running `brew update-reset`

**Comment by hubble63** [comment]
Now I got the messages:


```
brew update-reset
==> Fetching /usr/local/Homebrew...

==> Resetting /usr/local/Homebrew...
Branch 'master' set up to track remote branch 'master' from 'origin'.
Switched to and reset branch 'master'
Your branch is up to date with 'origin/master'.

==> Fetching /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask...

==> Resetting /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask...
Branch 'master' set up to track remote branch 'master' from 'origin'.
Reset branch 'master'
Your branch is up to date with 'origin/master'.

==> Fetching /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core...
remote: Enumerating objects: 254940, done.
remote: Counting objects: 100% (254940/254940), done.
remote: Compressing objects: 100% (70414/70414), done.
remote: Total 249785 (delta 186095), reused 242852 (delta 179331), pack-reused 
Receiving objects: 100% (249785/249785), 110.34 MiB | 11.92 MiB/s, done.
Resolving deltas: 100% (186095/186095), completed with 5085 local objects.
From https://github.com/Homebrew/homebrew-core
   c8f8fd0fa2..cc65a3cd27  master     -> origin/master

==> Resetting /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core...
Checking out files: 100% (6502/6502), done.
Branch 'master' set up to track remote branch 'master' from 'origin'.
Reset branch 'master'
Your branch is up to date with 'origin/master'.
```
What I have to do next?
Start again (?):

`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

**Reply by SMillerDev** [reply]
No, that's it. Brew should largely work now, except for the software that doesn't work on macOS 10.13 anymore.
